core/controllers/feedback.py

```diff
@@ -146,6 +146,17 @@ def post(self, exploration_id):
         self.render_json(self.values)
 
 
+class ReportHandler(base.BaseHandler):
```

Comment:
I don't think this belongs here, actually. Instead, I think you want it in core.controllers.reader. This is because the feedback module is for sending feedback to exploration authors, but in this case we are flagging an exploration for moderators to take a look at.

In addition, the class name and docstring are very vague; my first question on reading them is what the reports are meant to be about. Perhaps you could name the class ExplorationFlaggingHandler, and give a longer, more detailed docstring that explains what this handler is actually used for.